### ISSUE TITLE
Request: Do not preset tiles that have no imagery

### ISSUE THREAD

**Issue by bgirardot** [mapswipe/app-feedback#21]
I get tiles that have the no imagery icon (camera with line through it).

These can often be very large areas and triple tapping is a lot of tapping for something that I think can be detected in the backend code.

I think this was on downloaded tiles, not network mapping, not sure if that makes a difference or not.

**Comment by CloCkWeRX** [comment]
+1 to a hash or similar of tiles and auto marking if it is the no imagery icon - either by noticing when all 6 tiles are identical content; or a lookup of known no imagery checksums

**Comment by mathcass** [comment]
Responding to both @bgirardot and @CloCkWeRX, thanks for the suggestion. I know that @ivangayton has a working version of this in development that should resolve this, so the feedback for this feature is absolutely welcome and folks have taken some time to look at it. We hope to push up some updates sometime soon. 

As always, thank you for your efforts mapping.

**Comment by ericboucher** [comment]
Was that done @mathcass ?

**Comment by mathcass** [comment]
No, this hasn't been implemented @ericboucher. I think the work that Ivan did lives in the old NodeJS backend code but it may be possible to incorporate that in the new Python workers.
